### ISSUE TITLE
Stations with more than on Word return no data

### ISSUE THREAD

**Issue by MystifHA** [FaserF/ha-deutschebahn#44]
### The problem

Connections that contain stationnames with two or more words,  like for example Nürnberg-Stein

![image](https://github.com/user-attachments/assets/1951a9cb-afa0-474b-94d6-9a9831879b0d)
return no values.
How can i fix this when creating the connection ?

I think it has something to do with the underscores.

Thanks a lot !

### What version of Home Assistant Core has the issue?

2024.7.3

### What version of DeutscheBahn Integration has the issue?

3.0.1

### What type of installation are you running?

Home Assistant OS

### Example YAML snippet

```yaml
Example Sensor:
sensor.heilsbronn_to_nurnberg_stein
```

**Comment by FaserF** [comment]
This seems to be the same issue as #38 
I have prepared a fix. But it has to be merged into the Schiene python Module to be fixed

**Comment by MystifHA** [comment]
I have no Problem with Nürnberg. Connections to Nürnberg deliver correct results. Only if the stationname consists of two or more words it delivers no results.

**Comment by fuslwusl** [comment]
Same problem here. Connections with ßöäü do not work anymore since v3.0.2. version 3.0.1 works!

**Comment by FaserF** [comment]
> Same problem here. Connections with ßöäü do not work anymore since v3.0.2. version 3.0.1 works!

This is not true. In general those stations work. I have also tested it for example with "München-Pasing" to "Würzburg" which just works fine. I have no clue why only some stations are not working right now.

```
2024-10-13 15:40:17.282 DEBUG (MainThread) [custom_components.deutschebahn.sensor] Update the connection data for 'München-Pasing' 'Nürnberg-Stein'
2024-10-13 15:40:17.498 DEBUG (SyncWorker_59) [custom_components.deutschebahn.sensor] Fetched raw data: []
```
---> No issue with the integration, more with the dependency schiene, maybe my PR will help for this here too https://github.com/kennell/schiene/pull/39

"München-Pasing" to "Nürnberg-Stein" for example does not work.
